SQF/dayz_code/compile/player_switchModel.sqf

```diff
@@ -9,8 +8,9 @@ disableSerialization;
 _position = player modeltoWorld [0,0,0];
 _dir = getDir player;
 _currentAnim = animationState player;
-//_currentCamera = cameraView;
+_currentCamera = cameraView;
 _playerUID = getPlayerUID player;
+if (Z_SingleCurrency) then {_coins = player getVariable [Z_moneyVariable,0];};
```

Comment:
You want to do the getVariable and setVariable in player_humanityMorph with all the others. SwitchModel is called directly at login before this variable is set.